libcst/codemod/visitors/tests/test_add_imports.py

```diff
@@ -891,3 +891,36 @@ def test_import_in_docstring_module(self) -> None:
                 full_module_name="a.b.foobar", full_package_name="a.b"
             ),
         )
+
+    def test_import_in_docstring_module_with_docstring_not_as_the_top_line(
```

Comment:
nit: A "docstring" in the wrong location is not really a docstring at all, technically speaking. It's just a random string expression in a statement by itself, which is thrown away at runtime. So I would prefer to rename this test to `test_import_in_module_with_standalone_string_not_a_docstring`.

Reply:
Agreed, thank you

Reply:
Fixed